packages/query/src/index.ts

```diff
@@ -906,7 +906,13 @@ const generateQueryHook = async (
 
     const queryKeyFn = `export const ${queryKeyFnName} = (${queryKeyProps}) => [\`${route}\`${
       queryParams ? ', ...(params ? [params]: [])' : ''
-    }${body.implementation ? `, ${body.implementation}` : ''}];`;
+    }${body.implementation ? `, ${body.implementation}` : ''}] as ${
```

Comment:
You don't check to add the body type?

Reply:
> You don't check to add the body type?

Yes, I missed that one. However, I cannot seem to create valid specification that has this ```body.implementation``` . OpenAPI 3.0 and Swagger 2.0 both seem to not support ```requestBody``` on GET requests which create the query key function. Do you have an example which does generate a ```body.implementation``` for a query key function?

> Would be easier to do as const no?

Yes, I agree that would make it a bit more clean. I will make this return type string a separate const based on the ```queryParams``` and ```body``` for now